enzyme/Enzyme/CApi.h

```diff
@@ -133,6 +162,17 @@ LLVMValueRef EnzymeCreatePrimalAndGradient(
     size_t uncacheable_args_size, EnzymeAugmentedReturnPtr augmented,
     uint8_t AtomicAdd);
 
+/// Generates an augmented forward function based on the input.
+///
+/// Cached information will be stored on a Tape.
+/// Will usually be used in combination with EnzymeCreatePrimalAndGradient for
+/// reverse-mode-split AD.
+/// @param retType: DFT_DUP_ARG, DFT_CONSTANT
+/// @param constant_args: pointing to combinations of DFT_DUP_ARG and
+/// DFT_CONSTANT
+/// @param forceAnonymousTape: TODO
```

Comment:
Per https://enzyme.mit.edu/doxygen/classEnzymeLogic.html#a8e6ba5c54950f1ebcdfcbd7ce2a16083, `forceAnonymousTape forces the tape to be an i8* rather than the true tape`.